ni_measurementlink_service/measurement/service.py

```diff
@@ -181,9 +181,13 @@ def __init__(
             ui_file_paths=ui_file_paths,
         )
 
+        def process_annotations(value: object):
```

Comment:
This function doesn't process annotations. It processes the value part of one annotation.

"convert" or "escape" would be more descriptive than "process".

Suggestion: "convert_value"